docs/reference/aggregations/pipeline/avg-bucket-aggregation.asciidoc

```diff
@@ -1,44 +1,67 @@
 [[search-aggregations-pipeline-avg-bucket-aggregation]]
-=== Avg bucket aggregation
+=== Average bucket aggregation
 ++++
-<titleabbrev>Avg bucket</titleabbrev>
+<titleabbrev>Average bucket</titleabbrev>
 ++++
 
-A sibling pipeline aggregation which calculates the (mean) average value of a specified metric in a sibling aggregation.
-The specified metric must be numeric and the sibling aggregation must be a multi-bucket aggregation.
+A sibling pipeline aggregation which calculates the mean value of a specified
+metric in a sibling aggregation. The specified metric must be numeric and the
+sibling aggregation must be a multi-bucket aggregation.
 
 [[avg-bucket-agg-syntax]]
 ==== Syntax
 
-An `avg_bucket` aggregation looks like this in isolation:
-
 [source,js]
---------------------------------------------------
+----
 {
   "avg_bucket": {
-    "buckets_path": "the_sum"
+    "buckets_path": "the_sum",
+    "gap_policy": "skip",
+    "format": "#0.00;(#0.00)"
   }
 }
---------------------------------------------------
+----
 // NOTCONSOLE
```

Comment:
I think its probably helpful to spend the extra lines to turn this into a CONSOLE example. We'll get the test from it and readers will know where the agg sits in the request tree.

Reply:
Looking again, it really *is* a lot of lines. And we already say we'll have an example below. So I'm not really sure. I do like full examples, but this one would take up the whole screen......

Reply:
Thanks for the feedback @nik9000. I considered either converting this to a full example OR simply moving the example below up. However, it can be difficult to spot where exactly the `avg_bucket` agg begins/ends. We can do callouts, but that requires a bit more parsing on the part of the reader. I imagine that'll be a problem for most of our pipeline aggs.

Reply:
It is like you say, yikes. Do you think we could have the syntax block include a tagged region from the example below? That way at least it won't drift.

Reply:
That's a great idea! I implemented this with 105e3b162757da5e0ac7f164cb42f03200143f50 (and some cleanup with ee06c0dae9cc1d7fe3aa194fcec726f3578424a5).

The tagged region comments are now visible in the example. I don't think there is a substitution combo that lets you get rid of them. However, I don't think that's necessarily a bad thing. It does a good job of calling out the agg configuration in context.

Reply:
Is it worth adding callouts for the tagged regions? Just something to explain to the users that the tags indicate the agg configuration but aren't actually part of the example.